crates/nu-command/src/viewers/table.rs

```diff
@@ -351,7 +351,22 @@ fn handle_table_command(
             ctrlc,
             metadata,
         ),
-        x => Ok(x),
+        PipelineData::Value(v, ..) => {
+            // into_string() is used for serialising primitives in PipelineData::write_all_and_flush(),
+            // so the same is used here.
+            let str_representation = v.into_string("", config);
+
+            Ok(Value::String {
+                val: StyleComputer::from_config(engine_state, stack)
+                    .style_primitive(&v)
+                    .color_style
+                    .map(|e| e.paint(str_representation.clone()).to_string())
```

Comment:
```suggestion
                    .map(|e| e.paint(&str_representation).to_string())
```

I believe you can avoid a clone here by passing a reference to `str_representation`.